deploy-build.sh

```diff
@@ -29,6 +29,8 @@ shopt -s nullglob globstar
 #### functions
 ###
 
+BUILDS_DIR="tmp/builds"
```

Comment:
We could pull the shared stuff into a file called `globals.sh` which we `source` at the top of `publish-build.sh` and `deploy-build.sh` to make it explicit.

Reply:
Nice, that's a good call

Reply:
Neat. :zap:

Reply:
I decided to skip this for now since sourcing a relative file can be a bit complex, seems overkill to share one variable for now.

Side note: this might be a candidate for a node.js rewrite, could make it more maintainable.

Reply:
:+1: